scrapy/commands/check.py

```diff
@@ -73,6 +73,7 @@ def run(self, args, opts):
             spidercls.start_requests = lambda s: conman.from_spider(s, result)
 
             tested_methods = conman.tested_methods_from_spidercls(spidercls)
+            print('Checking spider %s...' % spidername)
```

Comment:
How is this different from the `print(spider)` at line 87/88, where not only spiders are printed but also their methods (`print('  * %s' % method)`)?

Reply:
> I see that, with the proposed verbose mode, extra information is printed about passing tests.
> 
> I’m used to verbose modes in test frameworks (e.g. pytest) providing additional details about failed tests, not about passed tests.
> 
> I feel printing additional information about passing tests could be a bit counterproductive: the more information you get about working tests, the harder it is to read information about failing tests. I would expect a verbose mode in `scrapy check` to work closer to how pytest works.
> 
> @kmike What’s your take?

I guess I'm looking at this from more of a beginner's point of view... I don't have much experience on pytest, but my idea was that if a user turns "verbose" on, he/she should be able to get more comprehensive information on how the crawler functioned. 

I can think of a couple of use cases of this proposed change:
1. When I try to write the check, I need to understand define a range, say 5 to 30 requests. If the website is somewhat dynamic, it might return a different number of requests each time. In this case, I think a verbose option should allow the user to know exactly how many requests they got. 
2. Check function should be a helpful tool for another developer to understand the crawler. If check can return more information, i.e., the fields that the crawler actually finds, it may help them to read the code faster. Of course they can read the source code as well, but scrapy check --verbose can act as a "minimal code" for them to see how things work.

Reply:
> How is this different from the `print(spider)` at line 87/88, where not only spiders are printed but also their methods (`print(' * %s' % method)`)?

 I guess this is not necessary, but if the crawler runs really slow for some reason, if you do the scrapy check command, then nothing shows up on the screen for the first 20 seconds... It might be nice for a user to see that the check command actually started.